cmd/storagenode/dashboard.go

```diff
@@ -80,7 +81,7 @@ func printDashboard(data *pb.DashboardResponse) error {
 	color.NoColor = !useColor
 
 	heading := color.New(color.FgGreen, color.Bold)
-	_, _ = heading.Printf("\nStorage Node Dashboard\n")
+	_, _ = heading.Printf("\nStorage Node Dashboard ( Node Version: %s)\n", version.Build.Version.String())
```

Comment:
This should come from the dashboard request.

Reply:
True, my plan is to add that in addition to the KAD Protobuf changes!